dotcom-rendering/playwright/lib/cmp.ts

```diff
@@ -27,7 +28,18 @@ const cmpAcceptAll = async (page: Page, rootIframe?: string): Promise<void> => {
 /**
  * Reject all on the Sourcepoint CMP banner
  */
-const cmpRejectAll = async (page: Page): Promise<void> => {
+const cmpRejectAll = async (
+	page: Page,
+	context: BrowserContext,
+): Promise<void> => {
+	// We must first allow reject all
+	const expires = new Date();
+	expires.setMonth(expires.getMonth() + 6);
+	await addCookie(context, {
```

Comment:
Use the `setAllowRejectAllCookie` function in allowRejectAll.ts instead of reproducing the logic

Reply:
unfortunately this is quite playwright-specific (it uses a `BrowserContext` object to set cookies)